arbnode/node.go

```diff
@@ -586,7 +590,9 @@ type Node struct {
 	ClassicOutboxRetriever *ClassicOutboxRetriever
 }
 
-type ConfigFetcher func() *Config
+type ConfigFetcher interface {
+	Get() *Config
+}
 type SequencerConfigFetcher func() *SequencerConfig
```

Comment:
Should SequencerConfigFetcher be also an interface for sake of consistency? then it probably will require wrapping ConfigFetcher in some struct providing Get() *SequencerConfig... can it be done nicer?